Resources/views/process/_form_complete.html.twig

```diff
@@ -57,7 +59,7 @@
                         <strong class="text-danger">{{ 'common.empty_field_value'|trans }}</strong>
                         <a class="ml-3" href="{{ path('process_edit', {id: process.id, referer: app.request.pathinfo}) }}">{{ 'common.actions.edit'|trans }}</a>
                     {% else %}
-                        <strong>{{ process.courtDecision|yes_no }}</strong>
+                        <strong>{{ process.courtDecision|court_decision }}</strong>
```

Comment:
Please use something like `{{ process.courtDecision ? 'court_decision.yes'|trans : 'court_decision.no'|trans }}`here! It is much better!